### PR TITLE
WP-5831 Prevent page zoom

### PR DESCRIPTION
**Pull request by pavlosamok**
### Description

Pinch-zooming in and out on the graph was leading to the whole page zooming in and out. The goal was to prevent such behavior.

It is usually done by intercepting the 'zoom' event and preventing its default behavior. The 'zoom' event for pinch-zoom is the 'wheel' event with truthy ctrlKey property.

Our version of react-diagrams already has the onWheel handler with some custom logic. Preventing the default behavior from inside of this handler wasn't working - it is considered a passive event listener by Chrome. Some details on this can be found [here](https://github.com/facebook/react/issues/19651)

I haven't found any react-specific ways to make an event handler active. As a workaround, I have added an active vanilla js event listener on the graph element.

I have also checked how [React Flow](https://reactflow.dev/) handling this situation. This library prevents pages from zooming in and out too and seems to be using the same approach with native preventDefault.

### JIRA Ticket

[WP-5831](https://varicent.atlassian.net/browse/WP-5831)

### CODE REVIEW

File: src/widgets/DiagramWidget.tsx [hunk 1]
```diff
@@ -667,6 +673,12 @@ export class DiagramWidget extends BaseWidget<DiagramProps, DiagramState> {
 		);
 	}
 
+	preventPageZoom(event) {
+		if (event.ctrlKey) {
```
Comment (jsweetman) [inline src/widgets/DiagramWidget.tsx:677]: ctrlKey? Is this true when you're pinch zooming?

**Review by jsweetman — CHANGES_REQUESTED** [review]
I think you need to run `yarn build:ts:prod` and then commit the changes in the dist folder as well (I know not ideal), see ui/README.md. Other than that looks good.

File: src/widgets/DiagramWidget.tsx (hunk 1 above)
Comment (pavlosamok) [reply src/widgets/DiagramWidget.tsx:677]: Yep, was a bit surprised too : )

**Comment by pavlosamok** [comment]
> I think you need to run `yarn build:ts:prod` and then commit the changes in the dist folder as well (I know not ideal), see ui/README.md. Other than that looks good.

Ah, I see. Thanks, I'll correct

**Review by jsweetman — APPROVED** [review]
Looks good

**Pull request merged**